src/schema/rules/tabular_data/pet.yaml

```
@@ -28,6 +28,7 @@ BloodPlasma:
     - suffix == "blood"
     - extension == ".tsv"
     - '"PlasmaAvail" in sidecar'
+    - sidecar.PlasmaAvail == true
```

Would this be any more efficient, since an absent `PlasmaAvail` will return `null`?

```suggestion
    - sidecar.PlasmaAvail == true
```